contracts/optimistic-ethereum/OVM/precompiles/OVM_SequencerEntrypoint.sol

```diff
@@ -8,6 +8,15 @@ import { Lib_SafeExecutionManagerWrapper } from "../../libraries/wrappers/Lib_Sa
 
 /**
  * @title OVM_SequencerEntrypoint
+ * @dev The Sequencer Entrypoint is a predeploy which, despite its name, can in fact be called by 
+ * any account. It accepts a more efficient compressed calldata format, which it decompresses and 
```

Comment:
Heh, good catch!  I wonder if there's any problems with anyone being able to call this; I don't think so.  Nobody should want to, since it would mean the fee sent by the ECDSAAccount gets sent to this address instead of their address.

Reply:
No, I don't think so. This actually came up in a conversation we had... 

But I'm going to consider that out of scope. :)